boards/arm/nucleo_l552ze_q/nucleo_l552ze_q.dts

```diff
@@ -32,3 +32,7 @@
 	current-speed = <115200>;
 	status = "okay";
 };
+
+&usart3 {
+	pinctrl-0 = <&usart3_tx_pd8 &usart3_rx_pd9>;
```

Comment:
Is there a reason to dissociate `pinctrl-0` defintion from `status = "okay"` ?
Why not moving `pinctrl-0` to file **nucleo_l552ze_q-common.dtsi** ?